parachain/runtime/snowbase/src/weights/ethereum_beacon_client.rs

```diff
@@ -0,0 +1,39 @@
+//! Autogenerated weights for ethereum_beacon_client
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-10-21, STEPS: `10`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("/tmp/snowbridge/spec.json"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/snowbridge
+// benchmark
+// pallet
+// --chain
+// /tmp/snowbridge/spec.json
+// --execution=wasm
+// --pallet
+// ethereum_beacon_client
+// --extrinsic
+// *
+// --steps
+// 10
+// --repeat
+// 10
+// --output
+// runtime/snowbase/src/weights/ethereum_beacon_client.rs
+// --template
+// templates/runtime-weight-template.hbs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for ethereum_beacon_client.
+pub struct SnowbridgeWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> ethereum_beacon_client::WeightInfo for SnowbridgeWeight<T> {	fn sync_committee_period_update() -> Weight {
+		(48_498_137_000 as Weight)			.saturating_add(T::DbWeight::get().reads(4 as Weight))			.saturating_add(T::DbWeight::get().writes(3 as Weight))	}	fn import_finalized_header() -> Weight {
```

Comment:
Formatting seems to be a bit off.  Is missing some newlines or something like that.

Reply:
Yeah, that's from the tilde (`~#each`) in the Handlebars template (it strips whitespace), like [here](https://github.com/Snowfork/snowbridge/blob/main/parachain/module-weight-template.hbs#L40). I noticed our template `module-weight-template.hbs` used it and so I also used it for the [runtime-weight-template.hbs](https://github.com/Snowfork/snowbridge/pull/701/files#diff-2b8a58c029afebd7be9c95411a9f5e49b8b5915babee07021a7de98677a277cc) because I thought there might be reason for it (but it did look very weird). 

I'll go ahead and remove those from both templates and regenerate the weights.